platform/test/spec/testEducationPlatformAppSpec.js

```diff
@@ -169,91 +169,4 @@ describe("EducationPlatformApp", () => {
             })
         })
     })
-
-
-    describe("notification()", () => {
```

Comment:
What's happened to all of these tests? Wouldn't it make sense to move them to the PlatformUtil tests spec?

Reply:
Yes, these were moved but the file was missing from the commit.  Fixed in b4d4f4a293d734496827ac955323ea46d437b208